src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -1501,3 +1501,89 @@ ActStatus ActionHandlers::handle_s1_rel_req_during_attach(ControlBlock& cb)
 
     return ActStatus::PROCEED;
 }
+
+#define S10_FEATURE
```

Comment:
Use makefile to enable/disable feature

Reply:
Sure, we are planning to do it that way itself.